pkg/cmd/render/render.go

```diff
@@ -559,8 +562,9 @@ func (t *TemplateData) setSingleStackIPv6(serviceCIDR []string) error {
 	return nil
 }
 
-func (t *TemplateData) setComputedEnvVars(platform string) error {
-	envVarMap, err := getEtcdEnv(platform, runtime.GOARCH)
+func (t *TemplateData) setComputedEnvVars(platform string, installConfig map[string]interface{}) error {
+	envVarData := &envVarData{platform, runtime.GOARCH, installConfig}
```

Comment:
Might not be a big deal right now, but in case the envVarData structure changes, I would suggest adding the structure member keys here.

Reply:
done